Velaptor/Factories/PathResolverFactory.cs

```diff
@@ -17,37 +20,87 @@ namespace Velaptor.Factories
-            atlasJSONDataPathResolver ??= new AtlasJSONDataPathResolver(IoC.Container.GetInstance<IDirectory>());
+        /// <returns>The resolver to texture content.</returns>
+        [SuppressMessage("ReSharper", "UnusedMember.Global", Justification = "Left here for future development.")]
+        public static IPathResolver CreateTextureAtlasPathResolver() =>
+            textureAtlasPathResolver ??= new AtlasTexturePathResolver(IoC.Container.GetInstance<IDirectory>());
+
+        /// <summary>
+        /// Creates a path resolver that resolves paths to fonts in the application's content directory.
+        /// </summary>
+        /// <returns>The resolver instance.</returns>
+        [SuppressMessage("ReSharper", "UnusedMember.Global", Justification = "Left here for future development.")]
+        public static IPathResolver CreateContentFontPathResolver() =>
+            contentFontPathResolver ??= new ContentFontPathResolver(IoC.Container.GetInstance<IDirectory>());
+
+        /// <summary>
+        /// Creates a path resolver that resolves paths to fonts in the systems font directory.
+        /// </summary>
+        /// <returns>The resolver instance.</returns>
+        public static IPathResolver CreateSystemFontPathResolver()
+        {
+            if (Platform.CurrentPlatform == OSPlatform.Windows)
+            {
+                return CreateWindowsFontPathResolver();
+            }
+
+            throw new NotImplementedException("Currently loading system fonts is only supported on Windows.");
+        }
 
         /// <summary>
         /// Creates a path resolver that resolves paths to font content.
         /// </summary>
         /// <returns>The resolver to atlas content.</returns>
-        public static IPathResolver CreateFontPathResolver() =>
-            fontPathResolver ??= new FontPathResolver(IoC.Container.GetInstance<IDirectory>());
+        public static IPathResolver CreateFontPathResolver()
+        {
+            var contentPathResolver = new ContentFontPathResolver(IoC.Container.GetInstance<IDirectory>());
+
+            return fontPathResolver ??= new FontPathResolver(
+                contentPathResolver,
+                CreateSystemFontPathResolver(),
+                IoC.Container.GetInstance<IFile>(),
+                IoC.Container.GetInstance<IDirectory>(),
+                IoC.Container.GetInstance<IPlatform>());
+        }
 
         /// <summary>
         /// Creates a path resolver that resolves paths to sound content.
         /// </summary>
         /// <returns>The resolver to sound content.</returns>
+        [SuppressMessage("ReSharper", "UnusedMember.Global", Justification = "Left here for future development.")]
         public static IPathResolver CreateSoundPathResolver() =>
             soundPathResolver ??= new SoundPathResolver(IoC.Container.GetInstance<IDirectory>());
+
+        /// <summary>
+        /// Creates a path resolver that resolves paths to fonts in the windows font directory.
```

Comment:
Grammar
```suggestion
        /// Creates a path resolver that resolves paths to fonts in the Windows font directory.
```
Is this Windows as in the OS or just a window?